server/website/website/tasks/async_tasks.py

```diff
@@ -244,8 +243,7 @@ def calc_next_knob_range(algorithm, knob_info, newest_result, good_val, bad_val,
     if mode == 'lowerbound':
         next_config = {knob.name: next_value}
     else:
-        knobs = SessionKnob.objects.get_knobs_for_session(session)
-        next_config = gen_test_maxval_data(knobs, knob.name, next_value)
+        next_config = {knob.name: next_value}
```

Comment:
I found that setting the non-tested knobs to default values (mostly 0 values) can make the tested knob achieve the largest range. For example, db_cache_size is bounded by shared_pool_size. shared_pool_size cannot be smaller than 5% of the db_cache_size. (This percentage is not accurate. It is just my experience.) When setting shared_pool_size to its minimal value, we cannot get the true maximal value of db_cache_size, However, setting shared_pool_size to 0 leads to no restriction of db_cache_size, and we can get the correct maximal value.